packages/ember-auto-import/ts/webpack.ts

```diff
@@ -526,7 +535,10 @@ export default class WebpackBundler extends Plugin implements Bundler {
             return '[' + deps.map((d) => `'${d}'`).join(',') + ']';
           }
         );
-        writeFileSync(resolve(this.outputPath, assetFile), outputSrc, 'utf8');
```

Comment:
According to the docs, `'utf8'` is the default value, so I just dropped it here to avoid more arity on `cachedWriteFileSync`